### PR TITLE
Upgrade app

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Upgrade to latest version.
```
Comment (TheoBrigitte) [inline CHANGELOG.md:12]: What is latest? Can we pin a version here please.

File: helm/grafana/Chart.yaml [hunk 2]
```diff
@@ -1,28 +1,27 @@
 apiVersion: v2
-appVersion: 7.3.5
+name: grafana
+version: 6.24.1
```
Comment (TheoBrigitte) [inline helm/grafana/Chart.yaml:3]: Why are we moving from `0.3.0` to `6.24.1` it seems a fairly large jump, what's the reasoning here ?

File: helm/grafana/Chart.yaml [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+appVersion: 8.4.2
```
Comment (TheoBrigitte) [inline helm/grafana/Chart.yaml:4]: This is a major bump? What are the breaking changes?

File: helm/grafana/Chart.yaml [hunk 4]
```diff
@@ -1,28 +1,27 @@
...
+  - name: Xtigyro
+    email: miroslav.hadzhiev@gmail.com
+  - name: torstenwalter
+    email: mail@torstenwalter.de
+engine: gotpl
+type: application
 annotations:
   application.giantswarm.io/team: team-atlas
-  config.giantswarm.io/version: 1.x.x
+  config.giantswarm.io/version: configure-grafana-app
```
Comment (TheoBrigitte) [inline helm/grafana/Chart.yaml:27]: Reminder to revert this before merging.

File: helm/grafana/Chart.yaml (hunk 2 above)
Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:3]: No reasonning apart that 0.3.0 is our own numbering :(

Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:3]: I need to put that to 0.4.0

File: helm/grafana/Chart.yaml (hunk 4 above)
Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:27]: Sure :)

File: CHANGELOG.md (hunk 1 above)
Comment (QuentinBisson) [reply CHANGELOG.md]: Done

File: helm/grafana/Chart.yaml (hunk 3 above)
Comment (QuentinBisson) [reply helm/grafana/Chart.yaml:4]: https://grafana.com/docs/grafana/latest/release-notes/release-notes-8-0-0/#breaking-changes Nothing really important

**Comment by QuentinBisson** [comment]
Tested on Gauss

**Review by TheoBrigitte — APPROVED** [review]

**Pull request merged**